src/pages/contact.js

```diff
@@ -50,6 +160,34 @@ class ContactUs extends Component {
     e.preventDefault()
     this.setState({ submitting: true })
 
+    const strippedState = Object.assign({}, this.state)
+    delete strippedState.name
+    delete strippedState.email
+    delete strippedState.message
+    delete strippedState.submitting
+    delete strippedState.triedSubmitting
+    delete strippedState.privacy
+
+    // if you tick and untick a field, it remains in state as 'false'; we want ot filter those out
+    const strippedFilteredState =
+      Object.keys(strippedState).length > 0
+        ? Object.keys(strippedState).filter(item => {
+            return strippedState[item]
+          })
+        : []
+
+    const branches =
+      strippedFilteredState.length > 0
+        ? strippedFilteredState.reduce((acc, current) => {
+            return (acc = [...acc, this.state[current].branch])
+          }, [])
+        : ['engineering']
```

Comment:
should this not be service?

Reply:
What do you mean?

Reply:
we have defined  two  branches for each interest in contentful, `service` and `community`, we shouldn't be introducing a new branch.